### ISSUE TITLE
sigaltstack: Cannot allocate memory

### ISSUE THREAD

**Issue by johnyf** [sagemath/cysignals#68]
```
$ pip freeze | ag cysignals
cysignals==1.6.7
$ python -c "import cysignals"
sigaltstack: Cannot allocate memory
```

Using:

- macOS Sierra 10.12.6
- Python 3.6.2

I do not get this error with `cysignals == 1.6.5` installed in the same environment.
Not sure whether this issue relates to #67.

**Comment by vbraun** [comment]
See also 
* https://trac.sagemath.org/ticket/21509
* https://groups.google.com/d/msg/sage-devel/sC6BEw9mg6M/HL7fFpPjAAAJ

**Comment by jdemeyer** [comment]
That is a known issue with 1.6.7 on OS X. Don't use that version.

**Comment by johnyf** [comment]
Thanks for addressing this issue. I installed ee6e81eaed82f87690da21be4ef7d3bc7b298473 and it works without this error.

**Comment by jdemeyer** [comment]
> I installed ee6e81e and it works without this error.

No no no no no! That version is still completely broken on OS X for a different reason.

**Comment by jdemeyer** [comment]
Fully fixed in version 1.6.8
